Extensions/Settings/Dnn.PersonaBar.SqlConsole/admin/personaBar/scripts/SqlConsole.js

```diff
@@ -355,6 +358,10 @@ define(['jquery',
                             rightData = rightData.toLowerCase();
                         }
 
+                        if(isNaN(leftData) && isNaN(rightData)) {
+                            return 0;
+                        }
```

Comment:
Uh oh, this won't work. The compared values can be numbers, text or boolean. The latter two will always be `NaN` and thus break sorting.